frontend/src/constants/queryFunctionOptions.ts

```diff
@@ -3,10 +3,6 @@ import { QueryFunctionsTypes } from 'types/common/queryBuilder';
 import { SelectOption } from 'types/common/select';
 
 export const metricQueryFunctionOptions: SelectOption<string, string>[] = [
-	{
```

Comment:
Removed this as we don't want to show the option in the dropdown and the functionality will be handled programmatically